--- a/zorp/policy.py.sample
+++ b/zorp/policy.py.sample
@@ -107,9 +107,9 @@ def zorp_http():
 	Service("intra_http", IntraHttp)
 	Service("intra_ftp", IntraFtp)
 
-	Rule(src_zone="site-net", dst_zone="internet", src_port=80, service="intra_http")
-	Rule(src_zone="site-net", src_port=21, service="intra_ftp")
-	Rule(src_zone="local", iface="eth0", dst_port=21, service="intra_ftp")
+	Rule(src_zone="site-net", dst_zone="internet", dst_port=80, service="intra_http")
+	#Rule(src_zone="site-net", src_port=21, service="intra_ftp")
+	#Rule(src_zone="local", iface="eth0", dst_port=21, service="intra_ftp")
 
 # zorp_plug instance
 def zorp_plug():
--- a/zorpctl/instances.conf.sample
+++ b/zorpctl/instances.conf.sample
@@ -11,5 +11,5 @@
 # of tabs! Otherwise zorpctl will stop working.
 
 #instance  arguments
-#zorp_http --verbose=5 --policy /etc/zorp/policy-http.py
-#zorp_plug --policy /etc/zorp/policy-plug.py
+#zorp_http --verbose=5 --policy /etc/zorp/policy-http.py --user root --group root
+#zorp_plug --policy /etc/zorp/policy-plug.py --user root --group root
