Index: zorp/zorpctl/zorpctl.conf
===================================================================
--- zorp.orig/zorpctl/zorpctl.conf	2014-07-22 11:39:46.000000000 +0200
+++ zorp/zorpctl/zorpctl.conf	2014-07-22 11:49:04.856996126 +0200
@@ -1,5 +1,4 @@
-#
-# This file is sourced by zorpctl and contains default settings
+s file is sourced by zorpctl and contains default settings
 # for various parameters.
 #
 [zorpctl]
@@ -26,13 +25,13 @@
 
 # whether to check /etc/zorp permissions
 CHECK_PERMS=1
-#ZORP_SYSCONFDIR=/etc/zorp
-#CONFIG_DIR_OWNER=root
-#CONFIG_DIR_GROUP=zorp
-#CONFIG_DIR_MODE=0750
+CONFIG_DIR=/etc/zorp
+CONFIG_DIR_OWNER=root
+CONFIG_DIR_GROUP=root
+CONFIG_DIR_MODE=0750
 
 # directory where Zorp stores its pidfiles
-#ZORP_PIDFILEDIR=/var/run/zorp
+PIDFILEDIR=/var/run/zorp
 
 # directory where Zorp executable exists
 #ZORP_LIBDIR=/usr/lib/zorp
@@ -40,6 +39,6 @@
 # set pidfile directory ownership according to the settings below (umask is
 # applied). 
 # DO not modify this, unless you know what you are doing.
-PIDFILE_DIR_OWNER=zorp
-PIDFILE_DIR_GROUP=zorp
+PIDFILE_DIR_OWNER=root
+PIDFILE_DIR_GROUP=root
 PIDFILE_DIR_MODE=0770
